05. Exercises/Exercise_1.md

```diff
@@ -11,6 +11,34 @@ Before starting this exercise, ensure you have:
 - **Two active GitHub accounts**
     - One acts as the repository owner
     - The other acts as the collaborator who receives the invitation
+- **Common error (first-time commit)**
+    - On your first commit, you may see an error message like: `"*** Please tell me who you are."`
+    - This happens because Git does not yet know your name and email.
+    - ![ex1_prerequisite_1.png](images/ex1_prerequisite_1.png)
+    - Configure them once globally with:
+      ```bash
+      git config --global user.name "Your Name"
+      git config --global user.email "you@example.com"
+      ```
+    - Or configure local for current repository with:
+      ```bash
+      git config user.name "Your Name"
+      git config user.email "you@example.com"
+      ```
+
+- **Verify setup**
```

Comment:
hmm good point, but this one should be in the 00. Init module after the User configuration